python/ParcelDelineation/udf_sobel_felzenszwalb.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import xarray
+from openeo.udf import XarrayDataCube
+from skimage import segmentation
+from skimage.filters import sobel
+from skimage import graph
+
+
+def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
```

Comment:
like the other UDF, can we also here avoid the use of XarrayDataCube and just use a DataArray?